src/fixed_width_ints.rs

```diff
@@ -18,3 +18,70 @@ pub type uint16_t = u16;
 pub type uint32_t = u32;
 #[deprecated(since = "0.2.55", note = "Use u64 instead.")]
 pub type uint64_t = u64;
+
+cfg_if! {
+    if #[cfg(all(target_arch = "aarch64",
+        any(target_os = "android", target_os="macos", target_os = "ios")))] {
```

Comment:
I would feel comfortable just doing `not(windows)` here. That way all non-Windows platforms automatically get support and this is confirmed by the static assert.